languages/ruby/exercises/concept/floating-point-numbers/.meta/design.md

```diff
@@ -0,0 +1,42 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to teach the student the concept of floating-point numbers and introduce them to the `while` loop.
+
+## Learning objectives
+
+- Know about the [Float][float-class] class and some of its methods.
+- Know how to write a `while` loop.
+
+## Out of scope
+
+- Parsing floating-point types from strings.
+- Converting floating-point types to strings.
+- Using standard or custom format strings.
+
+## Concepts
+
+The Concepts this exercise unlocks are:
+
+- `floating-point-numbers`
+- `while-loops`: know how to write a `while` loop.
+
+## Prerequisites
+
+This exercise's prerequisites Concepts are:
+
+- `numbers`: define numbers and apply arithmetic and boolean logic to them.
```

Comment:
```suggestion
- numbers: define numbers and apply arithmetic and boolean logic to them.
```

Reply:
Concepts are currently in "`" throughout this repo, so let's leave this as it is for now.